### ISSUE TITLE
On "Quick Start Guides", Recommended reading citations are not showing up

### ISSUE THREAD

**Issue by danblah** [ResetNetwork/recoding-tech#26]
See https://recoding-tech-v2.netlify.app/guide/disinformation/ and https://recoding-tech-2d607.sanity.studio/desk/guide;446fbe65-e86a-418e-aaa6-1569238b313e

**Comment by rudietuesdays** [comment]
Likely some style changes that didn't get pushed; will update!

**Comment by danblah** [comment]
Great, lemme know when you've pushed and ready for review!

**Comment by danblah** [comment]
Bump bump

**Comment by harumhelmy** [comment]
I'm on this @danblah ! Will have something for @rudietuesdays to review by tomorrow

**Comment by harumhelmy** [comment]
![giphy-1](https://user-images.githubusercontent.com/13320420/148293637-65291718-c66b-4415-83c6-6671b5e5b355.gif)

Sorry for the hold up with this one @danblah @gt150  -- @rudietuesdays and I went around in circles a little too long (actual picture of us shown above) trying to figure out how to make the Citations show up as a Carousel as shown in the wireframes. The way we decided to structure the data in the backend made it too complicated to do so. We decided to push up what we have -- the recommended reading are showing up on guides -- while still working out the logic that might allow us to display the readings in a carousel. We'll keep you posted.
